src/web/screens/teamMembers.tsx

```diff
@@ -28,21 +30,61 @@ function TeamMembers() {
   const onTeamMembersUpdated = useCallback(() => {
     reloader.revalidate();
   }, [reloader]);
+  const [showStatusPopup, setShowStatusPopup] = useState<boolean>(false);
+  const [statusPopup, setStatusPopup] = useState<StatusNotificationType>();
+  const [popupDuration, setPopupDuration] = useState<number>(3000);
+
+  const handleErrorPopup = (e: Error) => {
+    const hash = Object.hasOwn(e, 'errorHash') ? `: (${(e as ApiError).errorHash})` : '';
+    setStatusPopup({
+      type: 'Error',
+      message: `${e.message}${hash}`,
+    });
+    setShowStatusPopup(true);
+    setPopupDuration(10000);
```

Comment:
maybe we could change the `<StatusPopup>` component to decide the displayDuration based on the type? If we say Error should be 10000 but everything else is 3000? What do you think?

Reply:
I like this idea as a default behavior if no duration was passed to the component.  I changed it.